### PR TITLE
Using accessMode is required in this step of spec, Change also in ake…

### PR DESCRIPTION
**Pull request by MaorPaz**
Pleaase review if this configuration is required, @barucoh we talked about it, 
but i just wanted to make everyone else synced

### CODE REVIEW

File: akeyless-services/templates/persistentVolumeClaims.yaml [hunk 1]
```diff
@@ -9,8 +9,7 @@ metadata:
     app: {{ $.Values.containerName }}
     chart: {{ $.Chart.Name }}
 spec:
-  accessModes:
-  {{- $.Values.persistence.accessModes | toYaml | nindent 2 }}
+  accessModes: {{- $.Values.accessMode | toYaml | nindent 2 }}
```
Comment (barucoh) [inline akeyless-services/templates/persistentVolumeClaims.yaml:12]: This change is not needed, the code works.
Just need to enhance the error message.
Please keep the `accessModes` attribute under the `persistance` key.

**Review by barucoh — CHANGES_REQUESTED** [review]

File: akeyless-services/templates/persistentVolumeClaims.yaml (hunk 1 above)
Comment (MaorPaz) [reply akeyless-services/templates/persistentVolumeClaims.yaml:12]: done

File: akeyless-services/values.yaml [hunk 2]
```diff
@@ -127,7 +127,10 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-env: []
+env:
+  - name: VERSION
+    value: 1.31.2
+
```
Comment (barucoh) [inline akeyless-services/values.yaml:133]: Remove

**Review by barucoh — CHANGES_REQUESTED** [review]

File: akeyless-services/values.yaml (hunk 2 above)
Comment (MaorPaz) [reply akeyless-services/values.yaml]: done

File: akeyless-services/values.yaml [hunk 3]
```diff
@@ -127,7 +127,11 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-env: []
+
+#env:
+#  - name: VERSION
+#    value: 1.31.2
+
```
Comment (barucoh) [inline akeyless-services/values.yaml:134]: This should be removed, not commented out.
This code shouldn't be updated

**Review by barucoh — CHANGES_REQUESTED** [review]

File: akeyless-services/templates/persistentVolumeClaims.yaml [hunk 4]
```diff
@@ -9,8 +9,7 @@ metadata:
     app: {{ $.Values.containerName }}
     chart: {{ $.Chart.Name }}
 spec:
-  accessModes:
-  {{- $.Values.persistence.accessModes | toYaml | nindent 2 }}
+  accessModes: {{- $.Values.persistence.accessMode | toYaml | nindent 2 }}
```
Comment (liorhasson) [inline akeyless-services/templates/persistentVolumeClaims.yaml:12]: why?

File: akeyless-services/values.yaml (hunk 3 above)
Comment (MaorPaz) [reply akeyless-services/values.yaml]: Done

File: akeyless-services/templates/persistentVolumeClaims.yaml (hunk 4 above)
Comment (MaorPaz) [reply akeyless-services/templates/persistentVolumeClaims.yaml:12]: This step is required when setting the persistence volume,

File: akeyless-services/values.yaml [hunk 5]
```diff
@@ -127,8 +127,6 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-env: []
```
Comment (barucoh) [inline akeyless-services/values.yaml:130]: You weren't supposed to remove the `env` key completely, just leave it empty.
This could should not be changed.

**Review by barucoh — CHANGES_REQUESTED** [review]
In all honesty, I think this PR can just be closed as it doesn't have any relevant changes...

File: akeyless-services/values.yaml (hunk 5 above)
Comment (MaorPaz) [reply akeyless-services/values.yaml:130]: Got it, 
done

**Pull request closed without merging**